Add pgrouting binaries for bionic and documentation (#4472)
GeotrekCE/Geotrek-admin: docs/install/upgrade.rst

Starting from version 2.113.0, Geotrek now requires pgRouting. Please follow these steps before updating.

> If you use Ubuntu Bionic 18.04 on your database host (on same or external geotrek-admin host), you can follow the instructions below to install PgRouting on it.

Reviewer: ```suggestion
If you use Ubuntu Bionic 18.04 on your database host (on same or external Geotrek-admin host), you can follow the instructions below to install PgRouting on it.
```

4. Update Geotrek as usual

> **On a external database host: (Ubuntu >= 20.04 or docker with external database)**

Reviewer: ```suggestion
**On an external database host: (Ubuntu >= 20.04 or docker with external database)**
```

These instructions could be the same as previous if you use a debian like distribution on your database host.

1. Backup your database.
2. Install pgrouting version corresponding to your PostgreSQL version on your database server. (see `pgRouting documentation <https://docs.pgrouting.org/latest/en/index.html>`_).

> 3. Enable pgrouting extension in your geotrek database with your database superuser.

Reviewer: ```suggestion
3. Activate pgrouting extension in your Geotrek-admin database with your database superuser.
```

for Ubuntu bionic

Update PostgreSQL / PostGIS / PgRouting on Ubuntu Bionic
--------------------------------------------------------

.. warning::

    Ubuntu Bionic is already deprecated. We recommend you to install PostgreSQL on a dedicated server, with a most recent version of Ubuntu.
    If possible, on the same host or datacenter than your Geotrek-admin instance.
    If you can't, you can follow these instructions to upgrade PostgreSQL, PostGIS and PgRouting on Ubuntu Bionic with official PostgreSQL APT archive repository.
    The ultimate version published for Bionic is PostgreSQL 14, supported until November 12, 2026.

> I you have Postgresql < 14:

Reviewer: ```suggestion
If you have Postgresql < 14:
```
